app/templates/frameworks/_framework_actions.html

```diff
@@ -24,128 +24,80 @@
 {% endif %}
 
 {% if framework.status == 'open' %}
-<li class="browse-list-item">
-  <a class="browse-list-item-link"
-     href="{{ url_for('.supplier_details') }}">
-    <h2>1. Enter your company details</h2>
-  </a>
-  {% if not application_company_details_confirmed %}
-  <div class="browse-list-item-status-quiet">
-    <p class="browse-list-item-status-title">You must {% if supplier_company_details_complete %}confirm{% else %}add{% endif %} your organisation’s details</p>
-  </div>
-  {% else %}
-  {% if declaration_status == 'complete' and counts.complete %}
-  <div class="browse-list-item-status-happy">
-  {% else %}
-    <div class="browse-list-item-status-default">
-  {% endif %}
-    <p class="browse-list-item-status-title">
-      <strong>Your company details were saved</strong>
-    </p>
-  </div>
-  {% endif %}
-</li>
-
-<li class="browse-list-item">
-  {% if not application_company_details_confirmed %}
-    <h2>2. Make supplier declaration</h2>
-    <p class="instruction-list-item-box inactive">Can't start yet</p>
-  {% else %}
-    {% if declaration_status == 'unstarted' %}
-    <a class="browse-list-item-link" href="{{ url_for('.framework_start_supplier_declaration', framework_slug=framework.slug) }}">
-      <h2>2. Make supplier declaration</h2>
-    </a>
+ {% if application_made %}
+    {% if counts.draft %}
+      {{ dmAlert({
+        "titleText": "Your application is complete, and will be automatically submitted.",
```

Comment:
Suggest amending to: Your application is complete and will be submitted automatically.